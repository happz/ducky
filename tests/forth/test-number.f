: TEST
  DECIMAL
  123                            . CR
  [ HEX -7F ] LITERAL DECIMAL    . CR
  [ HEX 7FF7 ] LITERAL HEX       . CR
  [ HEX -7FF7 ] LITERAL 2 BASE ! . CR
;

