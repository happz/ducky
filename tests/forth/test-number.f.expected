123 
-127 
7FF7 
-111111111110111 
